Makefile

```diff
@@ -164,6 +166,14 @@ generate: controller-gen
 	go generate ./...
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+# Check and add (if missing) license header
+LICENSE_FILE=$(shell pwd)/LICENSE
+.PHONY: add-license 
+add-license: addlicense
+	@echo "[RUNNING] Checking/Adding license..."
+	@$(ADDLICENSE) -v -f $(LICENSE_FILE) api internal 2>&1
```

Comment:
I guess the downside of this approach is that this script now needs to be updated in the future if any other top-level directory is introduced and needs to be checked, other than `api internal`.

The other option is to go back to using `find` for this, unfortunately. But I think it can be done more cleanly than the first attempt was:

```bash
find . -type f -name "*.go" | xargs ./bin/addlicense -v -f LICENSE
```

@ebaron thoughts? I'd rather avoid the `find` and just let `addlicense` handle walking the tree, but I don't know how likely that `api internal` directory list is to change.

Reply:
Agreed, `addlicense` doesn't look like it's doing anything like marking directories as already-walked if you pass it a directory as well as some of its subdirectories, so it would end up iterating over the same files multiple times.

```bash
$ go list -f '{{.Dir}}' ./... | sed -e "s|^$(pwd)||" | cut -d/ -f 2 | uniq
api
internal
```

so

```bash
$ go list -f '{{.Dir}}' ./... | sed -e "s|^$(pwd)||" | cut -d/ -f 2 | uniq | xargs ./bin/addlicense -f LICENSE
```

Reply:
Tested and worked for me! This is nicee!